2024-02-04 15:15:29.940 [info] Started local extension host with pid 7235.
2024-02-04 15:15:29.983 [error] Extension 'github.copilot-chat' wants API proposal 'interactiveUserActions' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-04 15:15:29.984 [error] Extension 'github.copilot-chat' wants API proposal 'readonlyMessage' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-04 15:15:29.984 [error] Extension 'github.copilot-chat' wants API proposal 'chatVariables' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-04 15:15:29.984 [error] Extension 'ms-python.python' wants API proposal 'saveEditor' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2024-02-04 15:15:32.523 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:33.551 [info] [perf] Render performance baseline is 30ms
2024-02-04 15:15:33.594 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:15:37.604 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:04.899 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:07.768 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:10.510 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:13.460 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:16.533 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:17.941 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:21.087 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:22.211 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:24.718 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:30.692 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:31.860 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:37.887 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:46.157 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:49.091 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:54.613 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:56.284 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:57.780 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:58.815 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:16:59.896 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:01.940 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:03.347 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:07.872 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:10.845 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:18.812 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:17:20.263 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:15.566 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:33.647 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:40.391 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:42.638 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:43.966 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:49.387 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:53.125 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:19:54.634 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:06.417 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:37.302 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:20:38.513 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:22.418 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:28.678 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:43.141 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:21:44.317 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:05.717 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:07.794 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:33.180 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:43.047 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:23:52.081 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:08.289 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:09.532 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 15:24:47.923 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:41:57.913 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:53:07.317 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:54:04.023 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
2024-02-04 18:55:40.445 [error] jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.: Error: jedi.api.environment.InvalidPythonEnvironment: bin/python seems to be missing.
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339625
	at re (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2339919)
	at /home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334703
	at Immediate.<anonymous> (/home/elisena/.vscode/extensions/ms-python.python-2023.22.1/out/client/extension.js:2:2334723)
	at process.processImmediate (node:internal/timers:476:21)
